ecs-agent/api/resource/ebs_discovery_windows.go

```diff
@@ -0,0 +1,59 @@
+//go:build windows
+// +build windows
+
```

Comment:
Seems like the copyright message is missing. Could you please add something like the following? https://github.com/aws/amazon-ecs-agent/blob/master/agent/app/agent.go#L1